aws/awsutil/copy.go

```diff
@@ -49,7 +50,14 @@ func rcopy(dst, src reflect.Value, root bool) {
 		} else {
 			e := src.Type().Elem()
 			if dst.CanSet() && !src.IsNil() {
-				dst.Set(reflect.New(e))
+				if _, ok := src.Interface().(*time.Time); !ok {
```

Comment:
Have to check `*time.Time` explicitly, because doing this generically would be a breaking change.